lib/signs/utilities/source_config.ex

```diff
@@ -96,13 +96,14 @@ defmodule Signs.Utilities.SourceConfig do
   @enforce_keys [
     :stop_id,
     :direction_id,
+    :routes,
```

Comment:
The typespec also says this could be `nil`, which maybe now we don't expect it to be? (unless there are places we specifically set it to `nil`)

Reply:
Below this gets populated from the JSON in a way that makes `nil` possible. I don't think we actually (want to) do that, but until we can verify that assumption, probably best to leave it like that.